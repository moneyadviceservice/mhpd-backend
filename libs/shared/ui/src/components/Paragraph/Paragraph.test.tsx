import React from 'react';
import { render, screen } from '@testing-library/react';
import { Paragraph } from '.';

describe('Paragraph component', () => {
  it('renders correctly', () => {
    render(<Paragraph>Text goes here</Paragraph>);
    const paragraph = screen.getByTestId('paragraph');
    expect(paragraph).toMatchSnapshot();
  });
});
