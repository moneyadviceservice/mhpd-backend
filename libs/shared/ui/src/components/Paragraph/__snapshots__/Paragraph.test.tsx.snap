// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph component renders correctly 1`] = `
<p
  class="mb-4"
  data-testid="paragraph"
>
  Text goes here
</p>
`;
